crates/gpui/src/platform/linux/x11/window.rs

```diff
@@ -262,8 +262,6 @@ impl X11WindowState {
             .event_mask(
                 xproto::EventMask::EXPOSURE
                     | xproto::EventMask::STRUCTURE_NOTIFY
-                    | xproto::EventMask::ENTER_WINDOW
-                    | xproto::EventMask::LEAVE_WINDOW
```

Comment:
(sorry, forgot to submit this!)

Reply:
No, it's unrelated. Do you want me to submit it in a separate PR? It was introduced in the merge here: https://github.com/zed-industries/zed/pull/11235/commits/65d0cdd5f0753cf129a18014e01b0b2fa1b7e342. Not really an issue, but since we aren't actually using those events I figured they shouldn't be active.

Reply:
Eh, seems fine. Just didn't want any WIP-code-that-actually-doesnt-do-anything in here.